lib/dynamoid/document.rb

```diff
@@ -107,7 +107,7 @@ def build(attrs = {})
       def exists?(id_or_conditions = {})
         case id_or_conditions
           when Hash then where(id_or_conditions).first.present?
-          else !! find(id_or_conditions)
+          else !! find_by_id(id_or_conditions)
```

Comment:
Would like to see a spec for any behavior change we expect from switching to `find_by_id`.  This looks like it might be a significant change.

Reply:
@pboling `find` uses `find_by_id` internally. The only difference between them - `find` supports passing multiple ids. As far as I see `Document.exists?` accepts either one id or condition.

So `Document.exists?([1, 2, 3])` will not work. Do you think we should keep this behaviour?

Reply:
@pboling Looks like neither ActiveRecord nor Mongoid supports `exists?(array)` signature ([here](https://apidock.com/rails/ActiveRecord/FinderMethods/exists%3F) and [here](https://mongoid.github.io/old/en/mongoid/docs/querying.html)) and user can check multiple ids in the following way
`Document.exists?('id.in': [1, 2, 3])`

Reply:
Ok, I think the change is fine then.  Seems we did not have a spec for the old behavior of passing an array directly then.